epochStart/metachain/trigger.go

```diff
@@ -143,6 +143,11 @@ func (t *trigger) ForceEpochStart(round uint64) error {
 
 	t.currEpochStartRound = t.currentRound
 	t.isEpochStart = true
+	t.prevEpochStartRound = t.currEpochStartRound
```

Comment:
this is wrong. probably merge error. t.PrevEpochStartRound has to be on line 143, there is no need to duplicate the t.currEpochStartRound assignement.

Reply:
done.